ndb/src/google/cloud/ndb/_eventloop.py

```diff
@@ -345,25 +345,70 @@ def get_event_loop():
     raise exceptions.AsyncContextError()
 
 
-def add_idle(*args, **kwargs):
-    raise NotImplementedError
+def add_idle(callback, *args, **kwargs):
+    """Helper function.
 
+    Calls :method:`EventLoop.add_idle` on current event loop.
 
-def queue_call(*args, **kwargs):
-    raise NotImplementedError
+    Raises:
+        exceptions.AsyncContextError: If called outside of a context
+            established by :func:`~google.cloud.ndb.async_context`.
```

Comment:
We've often not document "transitive" exceptions elsewhere in `google-cloud-*`.  @dhermes Is documenting them preferred in `ndb`?

Reply:
I'm still learning the culture of this group--I'm happy to omit if that's consistent with prevailing style. Logically, to me, it made sense to document exceptions a caller is likely to encounter, whether or not they're raised in the body of the function itself. I don't have a strong feeling that it should be done one way or another, though.

Reply:
Yeah I'd say that's fine. Typically I avoid documenting transitive exceptions just because the true number of possible exceptions is massive (e.g. `MemoryError` or `KeyboardInterrupt` could occur) but when there are special exceptions worth pointing out I think it's perfectly acceptable.